.docs/user-guide/config.md

```diff
@@ -13,11 +13,111 @@ finally, delving into the details of more advanced settings.
 This section outlines the most common configuration scenarios encountered by
 `REX-Ray`'s users.
 
+The typical configuration is done by creating a configuration file at
+`/etc/rexray/config.yml`.  This file is reviewed when REX-Ray is being used
+interactively through CLI or started as a service.  See the advanced section
+for more information and options for configuration.
+
+### Example without Modules
+The following is an example configuration file that works for the VirtualBox
+driver.  Notice how the VirtualBox specific settings take place in two areas.
+The first is that we must declare `virtualbox` under `storageDrivers` and the second
+is that we have a rooted parameter called `virtualbox` to determine the driver
+specific parameters.  Each driver has documentation that includes details for
+the proper parameter name to use for configuration.
+
+```
+rexray:
+  logLevel: warn
+  storageDrivers:
+  - virtualbox
+virtualbox:
+  endpoint: http://10.0.2.2:18083
+  tls: false
+  volumePath: "/Users/your_user/Repos/vagrant/rexray/Volumes"
+  controllerName: SATA
+```
+
+### Example with Modules
+Modules introduce the ability to have a single `REX-Ray` instance present
+multiple personalities or volume endpoints. This is useful for many reasons.
+If the host you are configuring has access to multiple storage platforms, each
+platform can be configured using individual modules.  The example below
+demonstrates this with VirtualBox.
+
+```
+rexray:
+  logLevel: warn
+  storageDrivers:
+  - virtualbox
+  modules:
+    default-docker:
+      type: docker
+      desc: "The default docker module."
+      host: "unix:///run/docker/plugins/vb1.sock"
+      docker:
+        size: 1
+      virtualbox:
+        endpoint: http://10.0.2.2:18083
+        tls: false
+        volumePath: "/Users/your_user/Repos/vagrant/rexray/Volumes"
+        controllerName: SATA
+    virtualbox2:
```

Comment:
I believe this would be the good one to include there.

```
rexray:
  logLevel: info 
  storageDrivers:
  - virtualbox
  modules:
    default-docker:
      type: docker
      desc: "The default docker module."
      host: "unix:///run/docker/plugins/vb1.sock"
      docker:
        size: 1
    virtualbox2:
      type: docker
      desc: "The second docker module."
      host: "unix:///run/docker/plugins/vb2.sock"
virtualbox:
  endpoint: http://10.0.2.2:18083
  tls: false
  volumePath: "/Users/clintonkitson/Repos/vagrant/rexray/Volumes"
  controllerName: SATA
```